python_modules/libraries/dagster-dask/dagster_dask/data_frame.py

```diff
@@ -350,12 +364,22 @@ def _dataframe_loader_config():
         for read_from, read_opts in DataFrameReadTypes.items()
     }
 
-    return Selector(
+    return Shape(
         {
-            "read": Field(Selector(read_fields), is_required=False,),
+            "read": Field(
+                Selector(read_fields),
+                is_required=False,
+            ),
+            **{
+                util_name: util_spec["options"]
+                for util_name, util_spec in DataFrameUtilities.items()
+            },
             # https://github.com/dagster-io/dagster/issues/2872
             **{
-                field_name: Field(field_config, is_required=False,)
+                field_name: Field(
+                    field_config,
+                    is_required=False,
+                )
```

Comment:
should we add some sort of test or invariant to ensure we don't accidentally overwrite keys here or [1]

Reply:
Fair point, especially with the new keys I'm adding. I'll split out the addition of the deprecated keys and check that they don't overlap.

Reply:
Added two unit tests.